### DISCUSSION TITLE
Zoom Levels: 1,2,5,6,8,10,11,12,14,16,20,25,33,50,100%

### DISCUSSION THREAD

**Discussion by ghost** [Beep6581/RawTherapee#6698] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/126161988/222548705-990c6922-2b3d-448e-9687-6b01789b0995.png)

I don't want to open an issue for every thought, so I'd like to encourage this in the discussion. I thought my mouse wheel was kind of like that, but then I started clicking the magnifying glass and I get these zoom levels...

For me, they are completely useless and almost the wrong direction. My images are very large and I always view them at 100%. The next reduction is already 50% and then only 33% and then it will be minimal steps when everything is already visible on the screen.

Is it ok to request a change in steps?

Are there already various issues / considerations on this?

**Comment by Lawrence37** [comment, marked as answer]
There is an issue for it #6560

**Comment by ghost** [comment]
@Lawrence37 Thank you, this takes time to read :)
